tests/codeception/backend/functional/LoginCept.php

```diff
@@ -17,7 +17,6 @@
 $I->see('Password cannot be blank.', '.help-block');
 
 $I->amGoingTo('try to login with wrong credentials');
-$I->expectTo('see validations errors');
 $loginPage->login('admin', 'wrong');
 $I->expectTo('see validations errors');
```

Comment:
It is not duplicated `expectTo` is for code below not for code above, and we login 2 times with not correct credentials that is why it is used 2 times for description of the `user story`